server/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java

```diff
@@ -189,8 +197,8 @@ public void testGlobalAggregateNullRowWithoutMatchingRows() throws Exception {
         SQLResponse response = execute(
             "select sum(age), avg(age) from characters where characters.age > 112");
         assertThat(response.rowCount()).isEqualTo(1);
-        assertThat(response.rows()[0][0]).isNull();
-        assertThat(response.rows()[0][1]).isNull();
+        assertNull(response.rows()[0][0]);
+        assertNull(response.rows()[0][1]);
```

Comment:
I'm confused, this is the opposite of what we want?

Reply:
Apologies, I somehow reverted this file to a version before a previous PR, I'm reverting, thx for catching!